TACC/Core-CMS
# feat(css): accent color from purple to blue

## Overview

Change accent color in old (i.e. v3.10) styles.

<details>
Accent color in core-styles v0.13 (thus core-cms v3.10) was indeed purple.

But that was a bug, and Design refuses it be deployed to more sites.
</details>

## Related

Skipped.

## Changes

- **changed** accent colors in minified css

| Before (purple) | After (blue) |
| - | - |
| `--global-color-accent--x-light:#e3d7fd;--global-color-accent--light:#a387ed;--global-color-accent--normal:#784fe8;--global-color-accent--dark:#6039cc;--global-color-accent--x-dark:#3d189b;--global-color-accent--alt:#d2cce7;--global-color-accent--weak:#6039cc40;` | `--global-color-accent--x-light:#AAC7FF;--global-color-accent--light:#86aeff;--global-color-accent--normal:#003399;--global-color-accent--dark:#002266;--global-color-accent--x-dark:#001133;--global-color-accent--alt:#cbd4e7;--global-color-accent--weak:#00226640`

## Testing

1. Deploy on https://dev.cep.tacc.utexas.edu and https://pprd.mise.tacc.utexas.edu/.
2. Verify links are blue, not purple.

## UI

…
